test/1_stdlib/StringTraps.swift

```diff
@@ -42,11 +42,9 @@ StringTraps.test("endIndex/successor")
   .code {
   var s = "abc"
   var i = s.startIndex
-  ++i
-  ++i
-  ++i
+  i = i.advancedBy(3)
```

Comment:
Please keep the test code the exact equivalent.  The test name says it is testing the `successor()` function.